packages/node_modules/@webex/webex-core/src/interceptors/redirect.js

```diff
@@ -26,12 +26,13 @@ export default class RedirectInterceptor extends Interceptor {
    * @returns {Object}
    */
   onRequest(options) {
-    if (options.uri && options.uri.includes('https://idbroker')) {
-      return options;
-    }
-
-    if (options.uri.includes(this.webex.config.credentials.samlUrl) || options.uri.includes(this.webex.config.credentials.tokenUrl) || options.uri.includes(this.webex.config.credentials.authorizeUrl)) {
-      return options;
+    if (typeof options.uri === 'string') {
```

Comment:
If options is undefined, it will throw an error, let's add a check: 

```suggestion
    if (options.uri && typeof options.uri === 'string') {
```

Reply:
or you can do `onRequest(options = {}) {` on line 28